socorro/scripts/fetch_crashids.py

```diff
@@ -177,17 +183,31 @@ def main(argv=None):
     params['_columns'] = 'uuid'
 
     # Override with date if specified
-    datestamp = args.date
-    if 'date' not in params or datestamp != 'yesterday':
-        if datestamp == 'yesterday':
+    if 'date' not in params or args.date:
+        datestamp = args.date or 'yesterday'
+
+        if datestamp == 'today' or datestamp == 'now':
+            startdate = utc_now()
+        elif datestamp == 'yesterday':
             startdate = utc_now() - datetime.timedelta(days=1)
         else:
             startdate = datetime.datetime.strptime(datestamp, '%Y-%m-%d')
 
         enddate = startdate + datetime.timedelta(days=1)
+
+        startdate = startdate.strftime('%Y-%m-%d')
+        enddate = enddate.strftime('%Y-%m-%d')
+
+        if datestamp == 'now':
+            # If we want crashes for now, add some cache-busting bits to the
+            # url and also sort in reverse by date
+            startdate = '%sT00:00:%02d.000Z' % (startdate, random.randint(0, 59))
```

Comment:
Instead of randomizing the seconds when the datestamp is now, could we change the pattern passed to `strftime` to include hours, minutes, and seconds? Then we avoid the (admittedly low) risk of hitting a cached value by accident.

Reply:
I'm trying to make sure it works even if the client and server computers are out of sync time-wise. That's the big reason this code is wonky. But I think I can figure out a better way to do it without the chance of hitting cache that it does now.